### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by AlaaSaadeddin**
relates #53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,136 @@
...
+
+## Set up the app locally
+
+First clone this repo: `git clone https://github.com/GSG-G8/art-gallery.git`
+
+then run `npm i` and open new terminal then run `cd client` then `npm i` to install the dependencies for the app.
+
+### Database Setup
+
+In terminal type psql or pgcli if installed. Within psql/pcli enter the following commands:
```
Comment (Mu7ammadAbed) [inline README.md:57]: We created a `buildDatabase.sql` file which will do all these commands instead, So all these commands aren't really necessary

File: README.md [hunk 2]
```diff
@@ -1 +1,134 @@
...
+
+### As an admin
+
+- I can delete a artist/user/painting
+
+## Set up the app locally
+
+First clone this repo: `git clone https://github.com/GSG-G8/art-gallery.git`
+
+then run `npm i` and open new terminal then run `cd client` then `npm i` to install the dependencies for the app.
```
Comment (ranasobeid95) [inline README.md:53]:
```suggestion
then run `npm run project-setup` to install the dependencies for the app.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,134 @@
...
+## Solution
+
+Our site provides artists with the ability to sell their paintings at reasonable prices and the ability to communicate with the customer appropriately.
+
+## User Stories
+
+### As an artist
+
+- I can sign up / login and have my own profile
+- I can edit my profile data
```
Comment (mariamisa) [inline README.md:33]: * I can edit my information

File: README.md [hunk 4]
```diff
@@ -1 +1,134 @@
...
+- I can Check artworks for different artists
+- I can Search and filter art works
+- I can buy paintings and add it to my cart
+- I can Checkout from cart
+- I can add a review for the artist
+
+### As an admin
+
+- I can delete a artist/user/painting
+
```
Comment (mariamisa) [inline README.md:48]: ++ I can see all artist and  switch activate artist 
++ I can export all artist to excel file
++ I can see my budget on dashboard

**Pull request merged**